third_party/workspace.bzl

```diff
@@ -33,6 +34,9 @@ def tensorboard_workspace():
   tensorboard_typings_workspace()
   tensorboard_js_workspace()
 
+  # Set up TypeScript toolchain.
+  ts_setup_workspace()
```

Comment:
I verified that this is required. If this line is removed, I'll get the following error:

> ERROR: /usr/local/google/home/cais/tb-forks/tensorboard-3/tensorboard/components/tensor_widget/BUILD:61:1: every rule of type ts_devserver implicitly depends upon the target '@build_bazel_rules_typescript_devserver_deps//:node_modules/requirejs/require.js', but this target could not be found because of: no such package '@build_bazel_rules_typescript_devserver_deps//': The repository '@build_bazel_rules_typescript_devserver_deps' could not be resolved

Reply:
Got it. Kinda sad how ts_library works but this one doesn't.